### PR TITLE
home-page: add hero subtitle

### PR DESCRIPTION
**Pull request by djacu**
Add a subtitle indicating that Nix Planet is at SCaLE with a link back to the SCaLE site.

### CODE REVIEW

**Comment by djacu** [comment]
Put in a PR so y'all can suggest changes easily.
What do you think about the link back to the scale site?

File: templates/components/hero.html [hunk 1]
```diff
@@ -5,5 +5,10 @@
         </figure>
 
         <h1 class="hero-title">March 6th-7th, 2025 @ Pasadena, CA</h1>
+        <h2 class="hero-subtitle">
```
Comment (L-as) [inline templates/components/hero.html:8]: I think it should have dates for SCALE too to make it clear it's 4 days in total. A lot of people were confused last time and left without going to SCALE. At NixCon I was talking to a guy who thought that PlanetNix and SCALE were happening _at the same time_, which is wrong but not entirely obvious from the website.

Comment (djacu) [reply templates/components/hero.html:8]: Done!

**Review by L-as — APPROVED** [review]
I would like to see this merged if possible.

**Comment by djacu** [comment]
> I would like to see this merged if possible. (Before people misbuy their plane tickets.)

It's live friend :)

**Pull request merged**